modules/core/src/main/scala/lucuma/graphql/routes/Routes.scala

```diff
@@ -106,7 +106,7 @@ class HttpRouteHandler[F[_]: Temporal](service: GraphQLService[F]) {
 
   def toResponse(result: Either[Throwable, Json]): F[Response[F]] =
     result match {
-      case Left(err)   => BadRequest(service.format(err))
+      case Left(err)   => Ok(service.format(err)) // in GraphQL errors are reported in a 200 Ok response (!)
```

Comment:
Has to be done, but I suppose clients may need to be adjusted.

Reply:
@cquiroz @rpiaggio @toddburnside do you lads expect a 400 BAD REQUEST when there are errors in a GraphQL response? From my experiments with Clue it seems like it doesn't really expect this (i.e., it seems to correctly assume you always get a 200 OK back).